src/resources/parser/texture/basis.js

```diff
@@ -42,7 +42,11 @@ class BasisParser {
                         // remove the swizzled flag from the asset
                         asset.file.variants.basis.opt &= ~8;
                     }
-                    basisTranscode(url.load, result, callback, { unswizzleGGGR: unswizzleGGGR });
+                    var basisModuleFound = basisTranscode(url.load, result, callback, { unswizzleGGGR: unswizzleGGGR });
+
+                    if (!basisModuleFound) {
+                        callback('WARNING: Basis module not found. ' + asset.name + ' texture will not be loaded');
```

Comment:
This will be missleading, as often textures are called something like: `diffuse.jpg`, although it did not try loading actual jpg, but its `.basis` variant.

Reply:
Would `Texture asset 'diffuse.jpg' will not be loaded` be more clear or should be give the path to the basis texture `files/diffuse.basis`?

Reply:
`WARNING: Basis module not found. Asset "' + asset.name + '" basis texture variant will not be loaded.`